### ISSUE TITLE
VM parser does not correctly parse generic function parameters.

### ISSUE THREAD

**Issue by crelier** [dart-lang/sdk#27299]
See tests/language/generic_methods_generic_function_parameter_test.dart

parameter name expected
```
  foo<U>(bar<V>(T t, U u, V v)) => bar<int>(1, 2, 3);
               ^

```

**Comment by zoechi** [comment]
Is this supposed to work already?

```dart
foo/*<U>*/(bar<V>(T t, U u, V v)) => bar/*<int>*/(1, 2, 3);
```
should work.

**Comment by mhausner** [comment]
Fixed in 3e60d488429f6def6f0a6576edfb364a14a66a12.
